src/core/models.py

```diff
@@ -322,7 +363,7 @@ def certificate(self) -> str:
     @property
     def ca(self) -> str:
         """The root CA contents for the unit to use for TLS."""
-        return self.relation_data.get("ca", "")
+        return self.relation_data.get("ca-cert", "")
```

Comment:
**major** We need to make sure this is backward compatible or we update the field during upgrade, otherwise after upgrade, this won't work properly

Reply:
**IMPORTANT**
**HIGH PRIOROTY**
(Note to myself)